static/css/dashboard/dashboard-base.css

```
@@ -1,3 +1,45 @@
+@import url('https://fonts.googleapis.com/css2?family=Allerta+Stencil&display=swap');
```

Stylelint (17.7.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Allerta+Stencil&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Allerta+Stencil&display=swap'" (import-notation)

(import-notation)